workflows/diagnostics/fv3net/diagnostics/offline/_helpers.py

```diff
@@ -143,11 +150,32 @@ def insert_rmse(ds: xr.Dataset):
 
 
 def load_grid_info(res: str = "c48"):
+    if "ne" in res:
+        return load_grid_info_scream(res)
+    elif "c" in res:
+        return load_grid_info_fv3(res)
+    else:
+        raise ValueError(f"Unknown evaluation grid {res}.")
+
+
+def load_grid_info_fv3(res):
     grid = catalog[f"grid/{res}"].read()
     wind_rotation = catalog[f"wind_rotation/{res}"].read()
     land_sea_mask = catalog[f"landseamask/{res}"].read()
     grid_info = xr.merge([grid, wind_rotation, land_sea_mask])
-    return safe.get_variables(grid_info, GRID_INFO_VARS).drop_vars(
+    return safe.get_variables(grid_info, GRID_INFO_VARS_FV3).drop_vars(
+        "tile", errors="ignore"
+    )
+
+
+def load_grid_info_scream(res):
+    grid = catalog[f"grid/{res}"].read()
+    land_sea_mask = catalog[f"landseamask/{res}"].read()
+    r_earth = 6.376e6
+    grid.area.values = grid.area.values * r_earth ** 2  # convert from radian^2 to m^2
```

Comment:
This is a bit scary... do we expect user to remember to do this every time they load the "area" data for SCREAM? In practice this catalog item gets used a lot in notebooks and other places.